### ISSUE TITLE
careful-gaur cycle 32

### ISSUE THREAD

**Issue by CoffeeMakesMorningsBetter** [LearnersGuild/los#169]
### Expected vs. Actual

_What were you attempting to do? What actually happened? Please provide your project name and/or the cycle number if applicable._

I don't believe my stats were properly recalculated from last week

First go around Eugene and I had different hours understandable (40/33) 40%/60% contribution, I lost 32 ELO

Yesterday the hours were rectified 40/40 with same contribution and I lost 36 ELO. Can you explain why my ELO dropped further and why the drastic change. I think if math servers me well it should have went up. 

I would be grateful if you could look into it because that is a lot of ELO to lose based on 40/60 contribution with the different(same) billable hours.

Please advise

### Steps to Reproduce

_How can you reproduce the bug? Include as much detail as you can._

### Environment and Versions

_What versions of relevant external software were you using (browser, OS, etc.)?_

**Comment by jeffreywescott** [comment]
@CoffeeMakesMorningsBetter Elo points are awarded based on _effectiveness_. That is, for each hour you worked, how much of the overall project's contribution was a result of that hour. So, if you report _lower_ hours, Elo can potentially go up.

There is also what we call a "scaling factor". The scaling factor basically says that a 66.67% to 33.33% contirbution to be "the same as" a 100% to 0% contribution. In other words, if someone contributes twice as much as someone else, it works as if the person contributing less contributed nothing at all (WRT how the Elo points are awarded). Since the contribution was 60% to 40%, that's a _significant_ way toward "contributed twice as much".

Bottom line: your stats were computed properly.
